src/GraphQLParser/Exceptions/GraphQLParserException.cs

```diff
@@ -49,7 +51,7 @@ private static string HighlightSourceAtLocation(ReadOnlySpan<char> source, Locat
         int padLen = nextLineNum.Length;
         string[] lines = source
             .ToString()
-            .Split(new string[] { "\n" }, StringSplitOptions.None)
+            .Split(NewlineSeparator, StringSplitOptions.None)
```

Comment:
For .NET Standard 2.1 and .NET 6 TFMs we can use this overload:
https://learn.microsoft.com/en-us/dotnet/api/system.string.split?view=net-8.0#system-string-split(system-char-system-stringsplitoptions)

So we could do this:
```suggestion
#if NETSTANDARD2_1 || NET6_0_OR_GREATER
            .Split('\n', StringSplitOptions.None)
#else
            .Split(_newlineSeparator, StringSplitOptions.None)
#endif
```